CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@
 
 1. Clone the repo.
 2. Execute `yarn` at the root.
+3. Create a `.env` file at the root, based on the `.env.template` file but with actual values.
```

Comment:
Where do we get the actual value of the sled api key from to run locally?

Reply:
i'll give it to you if you ask nicely :)